integration/testutil.go

```diff
@@ -67,6 +67,13 @@ const (
 	dockerLibrary                = "public.ecr.aws/docker/library/"
 )
 
+const (
+	alpineImage = "alpine:latest"
+	nginxImage  = "nginx:latest"
+	ubuntuImage = "ubuntu:latest"
+	drupalImage = "drupal:latest"
```

Comment:
Is it possible to pin versions?

Reply:
It is possible, but other existing tests rely on most of these values. I think that should be a separate PR.

Reply:
Can you create an issue for that please?

Reply:
https://github.com/awslabs/soci-snapshotter/issues/248